packages/portal-proto/next.config.js

```diff
@@ -69,7 +69,9 @@ module.exports = {
     NEXT_PUBLIC_APP_VERSION: process.env.npm_package_version,
     // NEXT_PUBLIC_BUILD_SHORT_SHA is passed from gitlab to docker when docker is not run it tries to get it directly from git
     NEXT_PUBLIC_APP_HASH:
-      process.env.NEXT_PUBLIC_BUILD_SHORT_SHA || buildHash(),
+      process.env.npm_lifecycle_event === "dev"
+        ? buildHash()
+        : process.env.NEXT_PUBLIC_BUILD_SHORT_SHA,
```

Comment:
This does not appear to work when running in prod mode.  Here's the steps I too
1. in the portal-proto workspace, copy the dev configs to the prod configs to allow us to run prod mode locally: `cp .env.development .env.production`
2. from root of the project, build everything: `npm run build`
3. from the protal-proto workspace, run the prod server: `npm run start`

When I check the footer, the commit hash does not appear:
<img width="314" alt="image" src="https://github.com/NCI-GDC/gdc-frontend-framework/assets/57954607/cb599a5f-e610-4bfc-b39e-f90effaf8f73">